mage/docker/docker.go

```diff
@@ -0,0 +1,26 @@
+package docker
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+func ExtractRepoDigest(inspectOutput string) (string, error) {
```

Comment:
I'm fine with either waiting and using the magefiles in a separate PR, or doing it here. I am just letting you know in case you missed it that this function can now be used from `[get.porter.sh/magefiles/docker.ExtractRepoDigest](https://github.com/getporter/magefiles/blob/35bfc96f50a849aefe9ba1e1288eb393568bae85/docker/images.go#L10)`

Reply:
Oh shoot I thought I removed these, I took out all calls to this from the magefile so this can be deleted